test/unit/operations/get_more.test.ts

```diff
@@ -0,0 +1,146 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+
+import { Long } from '../../../src/bson';
+import { MongoRuntimeError } from '../../../src/error';
+import { GetMoreOperation } from '../../../src/operations/get_more';
+import { Aspect } from '../../../src/operations/operation';
+import { ReadPreference } from '../../../src/read_preference';
+import { Server } from '../../../src/sdam/server';
+import { ServerDescription } from '../../../src/sdam/server_description';
+import { Topology } from '../../../src/sdam/topology';
+import { ClientSession } from '../../../src/sessions';
+import { MongoDBNamespace } from '../../../src/utils';
+
+describe('GetMoreOperation', function () {
+  const ns = new MongoDBNamespace('db.coll');
+  const cursorId = Object.freeze(Long.fromNumber(1));
+  const options = {
+    batchSize: 100,
+    maxTimeMS: 500,
+    readPreference: ReadPreference.primary
+  };
+
+  describe('#constructor', function () {
+    const server = new Server(new Topology([], {} as any), new ServerDescription(''), {} as any);
+    const operation = new GetMoreOperation(ns, cursorId, server, options);
+
+    it('sets the namespace', function () {
+      expect(operation.ns).to.equal(ns);
+    });
+
+    it('sets the cursorId', function () {
+      expect(operation.cursorId).to.equal(cursorId);
+    });
+
+    it('sets the server', function () {
+      expect(operation.server).to.equal(server);
+    });
+
+    context('options', function () {
+      const optionsWithComment = {
+        ...options,
+        comment: 'test'
+      };
+      it('does not set the comment option if the server version is <4', () => {
+        const server = new Server(
+          new Topology([], {} as any),
+          new ServerDescription(''),
+          {} as any
+        );
+        server.hello = {
+          maxWireVersion: 8
+        };
+        const operation = new GetMoreOperation(ns, cursorId, server, optionsWithComment);
+        const expected = {
+          batchSize: 100,
+          maxTimeMS: 500,
+          readPreference: ReadPreference.primary
+        };
+        expect(operation.options).to.deep.equal(expected);
+      });
+
+      it('sets the comment option if the server version is >=4', () => {
+        const server = new Server(
+          new Topology([], {} as any),
+          new ServerDescription(''),
+          {} as any
+        );
+        server.hello = {
+          maxWireVersion: 10
+        };
+        const operation = new GetMoreOperation(ns, cursorId, server, optionsWithComment);
+        expect(operation.options).to.deep.equal(optionsWithComment);
+      });
+    });
+  });
+
+  describe('#execute', function () {
+    context('when the server is the same as the instance', function () {
+      const server = new Server(new Topology([], {} as any), new ServerDescription(''), {} as any);
+      const session = sinon.createStubInstance(ClientSession);
```

Comment:
I refactored these two tests so that the stubs are created inside the test, rather than in the contexts.  I chose to just create them in the tests because there wasn't much that was used between the two tests.